### PR TITLE
Fixes scalajp/2015#509. Translates KenjiDoi_1.html

### PR DESCRIPTION
**Pull request by kimutansk**
Fixes scalajp/2015#509. Translates KenjiDoi_1.html

### CODE REVIEW

File: _candidates_en/KenjiDoi_1.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+length: 15
+audience: Beginner
+language: Japanese
+twitter: doiken208
+github: doiken
+icon: https://avatars0.githubusercontent.com/u/6147573?v=3&s=400
+organization: CyberAgent, Inc.
+---
+<p>
+At movie ad network named Lodeo, 6 Scala beginners and 4 members experienced under half a year have developed.
```
Comment (okapies) [inline _candidates_en/KenjiDoi_1.html]: Using "video" instead of "movie" should be better.

Comment (okapies) [inline _candidates_en/KenjiDoi_1.html]: 原文: 動画アドネットワークLodeoではscala歴０の６名と半年以下4名が開発に携わりました。
訳案: Lodeo is the video ad network, developed by 6 absolute Scala beginners and 4 people who have less than a half year of Scala experience.

File: _candidates_en/KenjiDoi_1.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+In this session, I will talk how to develop by Scala in the situation, and the good practice and bad practice while comparing the project start timing between the project mid-term when 2 beginners joined.
+</p>
+<p>
+Themes are following.
```
Comment (okapies) [inline _candidates_en/KenjiDoi_1.html]: The themes are **the** following: (_"is/are the following:"_ is the idiomatic phrase)

File: _candidates_en/KenjiDoi_1.html [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+audience: Beginner
+language: Japanese
+twitter: doiken208
+github: doiken
+icon: https://avatars0.githubusercontent.com/u/6147573?v=3&s=400
+organization: CyberAgent, Inc.
+---
+<p>
+At movie ad network named Lodeo, 6 Scala beginners and 4 members experienced under half a year have developed.
+In this session, I will talk how to develop by Scala in the situation, and the good practice and bad practice while comparing the project start timing between the project mid-term when 2 beginners joined.
```
Comment (okapies) [inline _candidates_en/KenjiDoi_1.html]: 原文: その中でLodeoがどのようにscalaでの開発を進めたか、０スタートのプロジェクト初期と初心者２名を交えた中期で比較しつつ、良かった習慣、悪かった習慣について発表します。
訳案: In this session, I would like to share our experience of developing Lodeo in Scala by comparing early phase of the project to middle phase of it with 2 beginners, and introduce good/bad practices of the development.

File: _candidates_en/KenjiDoi_1.html [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
+name: Doi Kenji
+title: "In conclusion how Scala in the project.-Scala development by the beginner group-"
```
Comment (eed3si9n) [inline _candidates_en/KenjiDoi_1.html]: 原文: 結局、業務でつかってどうなの？～初心者集団のScala開発～
訳案: "Scala impressions: Scala development by a group of beginners"

File: _candidates_en/KenjiDoi_1.html (hunk 1 above)
Comment (eed3si9n) [inline _candidates_en/KenjiDoi_1.html]: 訳案: Lodeo, an ad network for videos, was developed by a group of Scala beginners: 6 total with zero experience, and with 4 with less than a half year of Scala experience.

File: _candidates_en/KenjiDoi_1.html (hunk 2 above)
Comment (eed3si9n) [inline _candidates_en/KenjiDoi_1.html]: 訳案: The following is the outline:

File: _candidates_en/KenjiDoi_1.html [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+---
+<p>
+At movie ad network named Lodeo, 6 Scala beginners and 4 members experienced under half a year have developed.
+In this session, I will talk how to develop by Scala in the situation, and the good practice and bad practice while comparing the project start timing between the project mid-term when 2 beginners joined.
+</p>
+<p>
+Themes are following.
+</p>
+<ul>
+<li>An approach from Java to Scala</li>
```
Comment (eed3si9n) [inline _candidates_en/KenjiDoi_1.html]: 原文: Java -> Scalaアプローチ
訳案: Converting from Java to Scala

File: _candidates_en/KenjiDoi_1.html [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+<li>Easy understandable comments of the Pull request's review comment</li>
```
Comment (eed3si9n) [inline _candidates_en/KenjiDoi_1.html]: 原文: PRレビューの「伝わる」コメント
訳案: Helpful tips on writing comments for the PR reviews

**Comment by okapies** [comment]
LGTM

**Pull request merged**